templates/CRM/Event/Form/Registration/ThankYou.tpl

```diff
@@ -115,7 +115,7 @@
                   <div class="clear"></div>
                 </div>
             {/if}
-            {if $contributeMode ne 'notify' AND $trxn_id}
+            {if $trxn_id}
```

Comment:
always assigned so if not empty should render

Reply:
gawd - I knew setting it something made up is bad!

Reply:
Hmm - but that would have always been broken for PaypalPro & no-one has said anything? Because PaypalPro is not notify but it does do delayed confirm for recurring...

Reply:
Yeah - I suspect no-one will care... it probably seemed important once